sql-gremlin/src/main/java/org/twilmes/sql/gremlin/adapter/converter/ast/nodes/operator/GremlinSqlBasicCall.java

```diff
@@ -83,6 +84,8 @@ public String getRename() throws SQLException {
                 // returns the formatted column name for aggregations
                 return ((GremlinSqlAggFunction) gremlinSqlOperator).getNewName();
             }
+        } else if (gremlinSqlOperator instanceof GremlinSqlBinaryOperator) {
+            return ((GremlinSqlBinaryOperator) gremlinSqlOperator).getNewName();
```

Comment:
Shall we also add AS support, e.g. - `SELECT age, (age < 18) as minor FROM person`?

Reply:
Fixed.

Reply:
Fixed.